packages/core/src/amazonqFeatureDev/session/session.ts

```diff
@@ -167,7 +145,6 @@ export class Session {
 
             // If approach was changed then we need to set it in the next state and this state
             this.state.approach = newApproach
```

Comment:
Is the `state.approach` still being used?

Reply:
removed!